src/js/components/SummaryCount.jsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * This component is a single value wrapped in a container that can be used
+ * to showcase important values such as summary counts. Add as many as needed.
+ * This example uses FlexBox to layout the components.
+ *
+ * Usage:
+ * import {SummaryCount} from './SummaryCount'
+ *
+ * const style = {
+ *   flexWrapper: {
+ *     display: 'flex',
+ *     flexDirection: 'row',
+ *     flexFlow: 'row wrap',
+ *     justifyContent: 'space-around'
+ *   }
+ * }
+ *
+ * <div style={style.flexWrapper}>
+ *   <SummaryCount
+ *     title='COUNT OF CATS'
+ *     value='2000'
+ *    />
+ *    <SummaryCount
+ *     title='COUNT OF DOGS'
+ *     value='4000'
+ *   />
+ * </div>
+ */
+import numeral from 'numeral'
+import {Paper} from 'material-ui'
+import VisualizationToolbar from './VisualizationToolbar'
+import React, {PropTypes} from 'react'
+
+const style = {
+  paper: {
+    height: 110,
+    marginBottom: 20,
+    textAlign: 'center'
+  }
+}
+
+export const SummaryCount = (props) => {
+  const {
+    title,
+    value
+  } = props
+  
+  const newValue = numeral(value).format('0,0')
+  
+  return (
+    <Paper style={style.paper}>
+      <VisualizationToolbar title={title} />
+      <div>
+        <h2>
```

Comment:
I think `<h2>` elements are block elements, so I think the parent div is useless here